src/support.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 24 24"><defs><path d="M10 0C4.477 0 0 4.477 0 10s4.477 10 10 10 10-4.477 10-10C19.994 4.48 15.52.006 10 0zm7.857 7.222a.208.208 0 01-.196.278h-3.214a.21.21 0 01-.178-.1 5.042 5.042 0 00-1.667-1.667.21.21 0 01-.102-.18V2.34a.21.21 0 01.278-.197 8.376 8.376 0 015.079 5.08v-.001zM10 13.333a3.333 3.333 0 110-6.666 3.333 3.333 0 010 6.666zM7.222 2.143a.21.21 0 01.278.197v3.213a.21.21 0 01-.1.179c-.68.416-1.25.987-1.667 1.666a.21.21 0 01-.18.102H2.339a.21.21 0 01-.172-.083.206.206 0 01-.026-.19 8.376 8.376 0 015.08-5.084zM2.143 12.778a.207.207 0 01.196-.278h3.214a.21.21 0 01.178.1c.416.68.987 1.25 1.667 1.667a.21.21 0 01.1.178v3.213a.208.208 0 01-.279.196 8.376 8.376 0 01-5.076-5.076zm10.635 5.08a.208.208 0 01-.278-.196v-3.214a.21.21 0 01.1-.178 5.042 5.042 0 001.667-1.667.21.21 0 01.178-.1h3.213a.208.208 0 01.193.278 8.376 8.376 0 01-5.073 5.077z" id="a"/></defs><use fill="#000" fill-rule="nonzero" xlink:href="#a" transform="translate(2 2)"/></svg>
```

Comment:
- Please remove unnecessary `<defs>` `</defs>`
- Remove the `id="a"`
- Remove `<use fill="#000" fill-rule="nonzero" xlink:href="#a" transform="translate(2 2)"/>`